docs/application/dotnet/tutorials/certificates/signing-your-application.md

```diff
@@ -0,0 +1,15 @@
+# Signing Application with Certificate
+
+After creating a Samsung certificate for your galaxy watch, ensure that the Visual Studio IDE uses that same certificate to sign your application. 
+
+To use the same certificate in Visual Studio, follow these steps:
+
+1. In the Visual Studio menu, select **Tools > Options**.
+
+    ![Tizen Certification](media/certification_guide26.png)
+2. In the **Options** window that appears, select **Tizen > Certification** from the left sidebar.
+3. Select the **Sign the .TPK file using the following option.** checkbox.
+4. Select **Use profile of Tizen Certificate Manager** from the drop-down list as highlighted in the image.
+5. In the **Profile** section, select the certificate profile that you have created, and click **OK**.
+	
+    ![Options](media/certification_guide27.png)
```

Comment:
I can see the image is captured from the Windows in Korean.

![image](https://user-images.githubusercontent.com/9958374/84098298-dddbff00-aa41-11ea-829a-8bd10ef47287.png)

As we are guiding to capture the images from the Windows in Eng, 
@hakkyu-samsung , would you please capture this image again from the Windows in Eng if possible?
In the Windows in Eng, the file path is marked with \\, not ￦ as in Windows in Korean.

Reply:
@mijin85cho 
Thanks for the suggestion.
This may take some time, since I have to set up Windows in Eng, and the dev environment.
I will upload it during this week.

Reply:
@hakkyu-samsung
Thank you.
I will update the changes with the new image.

Reply:
@Reni-90 

Providing the new images:
* https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/certification-guide26.png
* https://d3unf4s5rp9dfh.cloudfront.net/Tizen_doc/certification-guide27.png

Reply:
@hakkyu-samsung 
Thank you, I will update the images and the changes.